docs/src/main/sphinx/connector/googlesheets.rst

```diff
@@ -133,16 +148,16 @@ No other types are supported.
 
 SQL support
 -----------
+In addition to the :ref:`globally available <sql-globally-available>` and :ref:`read operation <sql-read-operations>` statements,
+this connector supports the following features:
 
-The connector provides :ref:`globally available <sql-globally-available>` and
-:ref:`read operation <sql-read-operations>` statements to access data and
-metadata in Google Sheets.
+* :doc:`/sql/insert`
+* :doc:`/functions/table`
```

Comment:
Lets leave this out here since this is not done in any other connector doc either.

Reply:
Removed the `* :doc:/functions/table` section